openstack/blockstorage/v3/extensions/quotasets/testing/fixtures.go

```diff
@@ -0,0 +1,57 @@
+package testing
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/gophercloud/gophercloud/openstack/blockstorage/v3/extensions/quotasets"
+	th "github.com/gophercloud/gophercloud/testhelper"
+	"github.com/gophercloud/gophercloud/testhelper/client"
+)
+
+const FirstTenantID = "555544443333222211110000ffffeeee"
+
+var successTestCases = []struct {
+	name, httpMethod, jsonBody, uriPath string
```

Comment:
nit: have explicit field definitions for each of these.

Reply:
I'll make that change in my next update.